tests/Core/Components/Checkbox/FluentCheckboxTests.FluentCheckbox_Default.verified.razor.html

```diff
@@ -1,4 +1,5 @@
-<fluent-field id="xxx" label-position="after">
-    <fluent-checkbox aria-label="My aria label" autofocus="" checked="true" indeterminate="false" id="xxx" name="xxx" shape="square" size="medium" slot="input"></fluent-checkbox>
-    <fluent-text as="span" size="200" slot="message"></fluent-text>
-</fluent-field>
+
+<fluent-field id="xxx" label-position="after" size="medium">
```

Comment:
Why the FluentCheckbox tests changed?

Reply:
The Field/label did not take the Size into account yet.

Reply:
I did not make a change in `FluentField`.  I added `slot` (==AdditionalAttribute) in `FluentCheckBox.razor`. So the rendered html is different.

Reply:
I actually made the change outside of this PR already but didn't update the verified files...doing that in `dev-v5` now